nvflare/app_common/widgets/streaming.py

```diff
@@ -34,6 +35,8 @@
 _LOG_EXCEPTION_EVENT_TYPE = "analytix_log_exception"
 _LOG_CRITICAL_EVENT_TYPE = "analytix_log_critical"
 
+_LOGGING_EVENT_TYPE = "analytix_logging"
```

Comment:
The _LOG_XXX_EVENT_TYPE definitions are no longer used. Please delete them here.